Backend-Challenges.md

```diff
@@ -48,6 +48,23 @@ Generally speaking, an NFT consists of following components:
 
 The goal of this challenge is to flesh out the constructor, have it accept the arguments passed in the test, and make it set the max supply inside the constructor to what was passed as an argument.
 
+## Challenge 6 - BaseURI, TokenURI
+
+Since `_baseURIextended` is private, we have implemented a getter function called `_baseURI()`. The ERC721 standard already has this function implementation, but we want it to behave differently for our use case so we have overridden it. Notice the `override` keyword added to the function.
```

Comment:
I have linked the openzeppelin ERC721 page in the section "Solidity libraries and openzeppelin" which I have later on moved to the challenge 5 section in https://github.com/Byont-Ventures/Hackathon/pull/4. I think it made sense to include it here because part of that challenge is to figure out why the smart contract won't compile 

(ERC721 is already imported but the contract itself has not been declared ERC721 yet)

So I think it is better to include https://eips.ethereum.org/EIPS/eip-721 in the section with the openzeppelin ERC721 API links